app/src/main/java/org/simple/clinic/home/patients/links/PatientsTabLinkEffect.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.home.patients.links
+
+sealed class PatientsTabLinkEffect
```

Comment:
Why is it named PatientsTabLinkEffect and why does it need a method of `OpenPatientLineListDownloadDialog`?